src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWork.tsx

```diff
@@ -0,0 +1,35 @@
+import { Serif } from "@artsy/palette"
```

Comment:
If this is already in an `ArtworkDetails` folder, it can probably just be `AboutTheWork.tsx` (ie- drop the prefix).

Reply:
I *believe* we specifically added the full prefixes when @alloy was doing data for ArtworkSidebar.  This way it is easier to correctly understand which data fragment goes with which component. Though I might be misremembering something. @alloy - could you please confirm the proper naming here?

Reply:
Correct 👍 https://artsy.slack.com/archives/CAZG82X4G/p1530019149000063